src/snap.ts

```diff
@@ -53,6 +74,7 @@ wallet.registerRpcMessageHandler(async (originString, requestObject) => {
       return await removeAsset(wallet, originString);
     case 'getChainHead':
       // temporary method
+      polkadotEventEmitter.emit(PolkadotEvent.OnBalanceChange, "100 KSM");
```

Comment:
@mpetrunic This is a temporary invocation, just to expose emitting events easily while testing.